### PR TITLE
GWY-2901: Add example of how to copy the sandbox plans to live account.

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: examples/CopyPlansExample.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    public $liveAPIKey;
+    /**
+     * @var string Sandbox API key
+     */
+    public $sandboxAPIKey;
+
+    /**
+     * @throws Exception
+     */
+    public function __construct()
```
Comment (slavcodev) [inline examples/CopyPlansExample.php]: I do not see arguments here, seem the example from docblock won't work.

Comment (vova07) [inline examples/CopyPlansExample.php]: For some reason, was pushed the one of old version.
I'll update it again.

File: examples/CopyPlansExample.php [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    /**
+     * Save live plan.
+     *
+     * @param object $data The plan data
+     *
+     * @throws HttpException
+     */
+    protected function saveLivePlan($data)
+    {
+        $plan = new Plan($data->id);
```
Comment (slavcodev) [inline examples/CopyPlansExample.php]: I wonder can we clone the given plan? And then we can add the typecast method argument.

File: examples/CopyPlansExample.php [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+     * Get sandbox plans list.
+     *
+     * @param int $limit Limit
+     * @param int $offset Offset
+     *
+     * @return array The plan list.
+     *
+     * @throws HttpException
+     */
+    protected function getSandboxPlans($limit = 50, $offset = 0)
```
Comment (slavcodev) [inline examples/CopyPlansExample.php]: The encapsulation of method which not change instance state make no sense, IMO.

**Pull request merged**